docroot/modules/custom/nvli_custom_search_api/nvli_custom_search_api.module

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Contains nvli_custom_search_api.module..
+ */
+
+/**
+ *
+ * @param type $doc_id
+ *   Document Id.
+ * @return array
+ *   Entity Id.
+ */
+function nvli_custom_search_get_nid($doc_id) {
+  $nid = db_query('SELECT n.entity_id FROM {node__field_solr_docid} n WHERE n.field_doc_id_value=:doc_id', array(':doc_id' => $doc_id))->fetchField();
```

Comment:
Please use database service to perform the query rather than doing it the old way. Also, If this is a helper function, should be created as a service & Database service should be injected in there.